contributors/design-proposals/selector-generation-from-kubectl.md

```diff
@@ -0,0 +1,174 @@
+Selector generation from kubectl
+==================
+
+# Goals
+
+Make `kubectl apply` work well with selector.
```

Comment:
So I skimmed through the proposal and it seems to target the `apply` use case in the client. It will most likely be superseded if `apply` is moved server-side. I need to read the selector generation proposal for Jobs.